### PR TITLE
Alvaro martin

### PR DESCRIPTION
**Pull request by Naipsas**
Exercise 01 doesn't work properly.
Exercise 02 works fine.
Exercise 03 works fine.

Readme from all three exercises haven't been written yet.

### CODE REVIEW

File: alvaro_martin/session-02/exercise-02/mysqldump-job.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: mysqldump
+  namespace: "exercise-02"
+spec:
+  schedule: "*/1 * * * *" #"0 0 * * * "
```
Comment (javsalgar) [inline alvaro_martin/session-02/exercise-02/mysqldump-job.yaml]: A bit too much, don't you think?

File: alvaro_martin/session-02/exercise-03/mariadb-statefulset.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      app: mariadb # has to match .spec.template.metadata.labels
+  serviceName: "mariadb"
+  replicas: 1 # by default is 1
+  template:
+    metadata:
+      labels:
+        app: mariadb # has to match .spec.selector.matchLabels
+# From here, it's a POD
+    spec:
+# To solve the non-root permission problem
```
Comment (javsalgar) [inline alvaro_martin/session-02/exercise-03/mariadb-statefulset.yaml]: You can also use the `securityContext` directive

File: alvaro_martin/session-02/exercise-02/mysqldump-job.yaml (hunk 1 above)
Comment (Naipsas) [reply alvaro_martin/session-02/exercise-02/mysqldump-job.yaml]: Indeed! It was just to check if was working correctly (was even going to put it smaller!).

File: alvaro_martin/session-02/exercise-03/mariadb-statefulset.yaml (hunk 2 above)
Comment (Naipsas) [reply alvaro_martin/session-02/exercise-03/mariadb-statefulset.yaml]: Sure! Learned the solution from Tomas' article so I didn't knew it until it was commented in class.

**Pull request merged**